### ISSUE TITLE
mesh_pos parameters in .yml file

### ISSUE THREAD

**Issue by hetolin** [noahcao/Pixel2Mesh#16]
Hi, thank you for sharing this code. I am using this code to train new categories.
But I am confused that the code in sharpenet.py:
line 57     `pts -= np.array(self.mesh_pos)`

why the ponitclouds data should minus `mesh_pos=[0, 0, -0.8]`?

I read the code of the official/tensorflow version, it seems that they did not do this step.

**Comment by ultmaster** [comment]
Thanks for asking.

The pointcloud data are located at (0, 0, -0.8) to be rendered, and as a tryout, we would like to see whether normalizing the data to (0, 0, 0) will stabilize the training.

Actually, they don't help much.

**Comment by hetolin** [comment]
thank you for you prompt reply. So if I delete `pts -= np.array(self.mesh_pos)`, it still work?

Is there some relationship between  0.8 and the data in rendering_metadata.txt(Euler-z, Euler-y,euler-z, distance, fov)?  Where does the 0.8 come from?

**Comment by ultmaster** [comment]
The mesh_pos is a 3d coordinate (x, y, z). It's part of the training tricks. Feel free to remove it.

It comes from observation and I don't remember anything about `rendering_metadata.txt`: what is it?

**Comment by zshyang** [comment]
Hi,
Do you know how to download the dataset and unzip it from the link below?
https://drive.google.com/open?id=131dH36qXCabym1JjSmEpSQZg4dmZVQid

**Comment by zshyang** [comment]
Hi,
Do you know how to use their provided YAML files and pretrained weights downloaded from here to do the evaluation?
To my understanding, I should use the downloaded meta folder. Then use the provide pretrained weights.
The command I use is 
```
python entrypoint_eval.py --name xxx --options experiments/default/tensorflow.yml --checkpoint datasets/data/pretrained/vgg16-p2m.pth 
```
But it seems that it only generated some meaningless meshes. And the score is really low.
I am not sure what I did wrong.
![Screenshot from 2020-09-22 02-08-41](https://user-images.githubusercontent.com/57812785/93863819-de0eac00-fc78-11ea-91b5-0235f5b4487b.png)
